src/cmd_argument.rs

```diff
@@ -232,3 +232,33 @@ where
         config.working_directory = Some(self.0.as_ref().to_owned());
     }
 }
+
+/// Arguments of type [`PathBuf`] are passed into the child process
+/// as arguments.
+///
+/// ```
+/// use cradle::*;
+/// use std::path::PathBuf;
+///
+/// let current_dir: PathBuf = std::env::current_dir().unwrap();
+/// cmd_unit!("ls", current_dir);
+/// ```
+impl CmdArgument for PathBuf {
+    #[doc(hidden)]
+    fn prepare_config(self, config: &mut Config) {
+        config.arguments.push(self.into_os_string());
```

Comment:
I didn't actually try this, but suggesting it for consistency.

```suggestion
        config.arguments.push(self.into());
```

Reply:
Yup, works.